src/hotspot/share/gc/shared/gc_globals.hpp

```diff
@@ -186,6 +186,9 @@
   product(bool, UseThirdPartyHeap, false,                                   \
           "Use the thrtd-party heap for allocation and garbage collection") \
                                                                             \
+  product(ccstr, ThirdPartyHeapOptions, NULL,                               \
```

Comment:
```suggestion
  product(ccstr, ThirdPartyHeapOptions, "",                                 \
```

I think it's better to set the default value as an empty string instead of a null pointer, in case some code forgets to check/assumes the validity.

Reply:
I have seen existing OpenJDK code that declares an option of the type `ccstr` uses `NULL` as its default value. So I just followed their pattern. For example, here:
https://github.com/mmtk/openjdk/blob/e1e420b3c4fd627218e45ec977972561c559acec/src/hotspot/share/gc/z/z_globals.hpp#L42
Besides, dealing with null pointers is a norm for C++ code, so it might not actually be an issue.

However, I can change this to an empty string if you think it is better. Please let me know.

Reply:
I did a search on the code base. You are right. `NULL` is used for `ccstr` while `""` is used for `ccstrlist`. Let's be consistent and stick with `NULL`. I guess it's useful to know whether an option is not set `NULL` or deliberately set to `""`.